tensorrt_llm/_torch/modules/fla/utils.py

```
@@ -0,0 +1,326 @@
+        return triton.runtime.driver.active.utils.get_device_properties(
+            tensor_idx)["multiprocessor_count"]
+    except BaseException:
+        _cpu_device_warning()
+        return -1
+
+
+@lru_cache(maxsize=None)
+def get_available_device() -> str:
+    try:
+        return triton.runtime.driver.active.get_current_target().backend
+    except BaseException:
```

Ruff (0.13.1)

Do not catch blind exception: `BaseException`

(BLE001)